internal/core/util_test.go

```diff
@@ -336,13 +288,15 @@ func (d *DrandTestScenario) SetMockClock(t *testing.T, targetUnixTime int64) {
 // AdvanceMockClock advances the clock of all drand by the given duration
 func (d *DrandTestScenario) AdvanceMockClock(t *testing.T, p time.Duration) {
```

Comment:
honestly this whole function is puzzling to me - I dunno why we don't share a single clock instance between all the nodes in the test scenario rather than trying to align them all. I suppose one argument is that there'll be zero drift and wouldn't reflect 'real life', though I think even with separate clocks there'd be no drift